LayoutTests/inspector/css/getMatchedStylesForNodeViewTransitionPseudoElements.html

```diff
@@ -0,0 +1,163 @@
+        document.querySelector("#change").classList.add("changed");
+    });
+}
+
+function test()
+{
+    let suite = InspectorTest.createAsyncSuite("CSS.getMatchedStyleForNode.ViewTransitionPseudoElements");
+
+    function addTestCase({name, description, selector, domNodeStylesHandler})
+    {
+        suite.addTestCase({
+            name,
+            description,
+            async test() {
+                let documentNode = await WI.domManager.requestDocument();
+                let nodeId = await documentNode.querySelector(selector);
+                let domNode = WI.domManager.nodeForId(nodeId);
+                InspectorTest.assert(domNode, `Should find DOM Node for selector '${selector}'.`);
+
+                let domNodeStyles = WI.cssManager.stylesForNode(domNode);
+                InspectorTest.assert(domNodeStyles, `Should find CSS Styles for DOM Node.`);
+                await domNodeStyles.refreshIfNeeded();
+
+                await domNodeStylesHandler(domNodeStyles);
+            },
+        });
+    }
+
+    addTestCase({
+        name: "CSS.getMatchedStyleForNode.ViewTransitionPseudoElements.Root",
+        description: "A dialog should have both the User Agent and authored `::backdrop` rules.",
+        selector: "html",
+        async domNodeStylesHandler(styles) {
+            InspectorTest.expectEqual(styles.pseudoElements.get(WI.CSSManager.PseudoSelectorNames.ViewTransition), undefined, "Expected no rules entry for selector `*::view-transition` before starting the view transition.");
+            InspectorTest.expectEqual(styles.pseudoElements.get(WI.CSSManager.PseudoSelectorNames.ViewTransitionGroup), undefined, "Expected no rules entry for selector `*::view-transition-group()` before starting the view transition.");
+            InspectorTest.expectEqual(styles.pseudoElements.get(WI.CSSManager.PseudoSelectorNames.ViewTransitionImagePair), undefined, "Expected no rules entry for selector `*::view-transition-image-pair()` before starting the view transition.");
+            InspectorTest.expectEqual(styles.pseudoElements.get(WI.CSSManager.PseudoSelectorNames.ViewTransitionOld), undefined, "Expected no rules entry for selector `*::view-transition-old()` before starting the view transition.");
+            InspectorTest.expectEqual(styles.pseudoElements.get(WI.CSSManager.PseudoSelectorNames.ViewTransitionNew), undefined, "Expected no rules entry for selector `*::view-transition-new()` before starting the view transition.");
+
+            await InspectorTest.evaluateInPage(`startViewTransition()`);
+            await styles.refresh();
+            InspectorTest.expectEqual(styles.pseudoElements.get(WI.CSSManager.PseudoSelectorNames.ViewTransition).matchedRules.length, 2, "Expected entry for selector `*::view-transition` after starting the view transition.");
+
+            // FIXME: These should use the correct values when adding support for named pseudo-elements. We'll also want to test different names individually.
```

Comment:
can please you create a bug for this work and link it here so that it's tracked?